mavros/msg/ActuatorControl.msg

```diff
@@ -0,0 +1,12 @@
+# raw servo values for direct actuator controls
+# 0: roll
+# 1: pitch
+# 2: yaw
+# 3: throttle
+# 4: flaps
+# 5: spoilers
+# 6: airbrakes
+# 7: landing gear
+
+Header header
+float32[] channels
```

Comment:
Also change [] → [8]. And last thing is find how get pointer from `boost::array<float, 8>`.